inspirehep/modules/hal/utils.py

```diff
@@ -207,8 +209,11 @@ def get_conference_record(record):
         972464
 
     """
-    return replace_refs(get_value(
-        record, 'publication_info.conference_record[0]', default=None), 'db')
+    replaced = replace_refs(get_value(record, 'publication_info.conference_record[0]'), 'db')
+    if replaced:
+        return replaced
+    else:
+        return default
```

Comment:
Commits should be *atomic*; while implementing a new feature one shouldn't modify other parts of the code, even if they realized that there's a bug or a missing feature.